app/src/main/java/edu/mit/mitmobile2/SoloMapActivity.java

```diff
@@ -477,7 +459,36 @@ public void onMapLoaded() {
         }
     }
 
-    public void updateMITMapView(MITMapView mapView) {
+    public void updateMITMapView() {
         mapView.updateStaticItems(mapViewExpanded);
     }
+
+    public void refreshMapInfoWindow() {
```

Comment:
I don't think we want to be setting a new InfoWindowAdapter each time... better to have the marker information updated from the stops